osu.Game/Screens/Edit/Compose/Components/EditorBlueprintContainer.cs

```diff
@@ -133,8 +131,8 @@ protected override bool OnDoubleClick(DoubleClickEvent e)
         protected override void SelectAll()
         {
             Composer.Playfield.KeepAllAlive();
-
-            base.SelectAll();
+            SelectedItems.Clear();
+            SelectedItems.AddRange(Beatmap.HitObjects);
```

Comment:
I'm not sure about forcing a full re-selection here. At least while the operation has some overhead, I'd hope a second select all call is a no-op..